Dockerfile

```diff
@@ -20,6 +20,7 @@ ADD packages/squid/src ./packages/squid/src
 ADD packages/squid/schema.graphql ./packages/squid
 
 RUN corepack enable
+RUN yarn install
```

Comment:
Did you have any issue with that?
We should remove this because this installs all the dependancies, also for the ui, which are not needed.

Reply:
I'll merge main into this branch, this should solve any issue you had.